doc/Overview/Mvux/ListFeeds.md

```diff
@@ -4,63 +4,63 @@ uid: Overview.Mvux.ListFeeds
 
 # What are list-feeds?
 
-A list-feed (`IListFeed`) is like a [feed](xref:Overview.Mvux.Feeds) which is stateless and keeps no track of changes, but is specialized for handling collections.
+A `ListFeed` (`IListFeed`) is like a [Feed](xref:Overview.Mvux.Feeds) which is stateless and keeps no track of changes but is specialized for handling collections.
```

Comment:
This should be the actual type `IListFeed`, or should remain a descriptive name eg list-feed,

Reply:
We should also be using **bold** instead of `monospace` for references to names of classes etc (although [this guidance](https://learn.microsoft.com/en-us/style-guide/developer-content/formatting-developer-text-elements) is a bit ambiguous)